javascript-source/handlers/subscribeHandler.js

```diff
@@ -397,6 +514,20 @@
             $.say($.whisperPrefix(sender) + $.lang.get('subscribehandler.massgiftsub.msg.set'));
         }
 
+        /*
+         * @commandpath massanongiftsubmessage [message] - Set a message for resubscribers.
```

Comment:
Command description update here too.

update to message to `Set a message for mass anonymous gifting alerts.`